lib/emque/producing/message/message.rb

```diff
@@ -130,19 +128,27 @@ def to_json
         Oj.dump(data, :mode => :compat)
       end
 
-      def publish(publisher=nil)
-        publisher ||= Emque::Producing.publisher
-        log "publishing...", true
+      def publish(publishers=[:rabbitmq])
         if valid?
-          log "valid...", true
+          # logger.info uuid, topic type
```

Comment:
@charlie22c what do you think of something like this? If you set `publish_messages` to false, it would log but not publish. could be controlled by the logging level.